Cedexis/description.html

```diff
@@ -0,0 +1,46 @@
+<html>
+  
+  <head>
+    <link rel="stylesheet" type="text/css" href="preview.css">
+  </head>
+  
+  <body>
+    <!--<%def name="title()">Cedexis</%def>-->
+    <div class="sectiontitle" id="previewtitle"><h2>Cedexis</h2></div>
+    <div class="saas_tile" id="tile">
+      <div class="saas_avatar saas_in">
+        <img src="Images/200x128.png">
+      </div>
+      <div class="saas_content">
+        <p class="saas_name saas_in">Cedexis</p>
+        <div class="saas_controls">
+          <button class="meh">
+            <span class="js-icon text-success">+</span>
+            <span class="saas_msg" data-state_msg="Available" data-action_msg="Install">Available</span>
+          </button>
+        </div>
+        <div class="saas_description saas_in" id="tiledescription">Manage Openmix traffic with Datadog monitors and alerts.</div>
+      </div>
+    </div>
+    
+    <div class="integration_content" id="int-overview">
+      <h2>Overview</h2>
+      <p>Connect Datadog to Cedexis Openmix to:</p>
+      <ul>
+        <li>Manage traffic using Datadog monitors to measure the health of routing endpoints</li>
+        <li>Trigger immediate traffic changes from  Datadog alerts</li>
+      </ul>
+      Cedexis Openmix manages global application and content delivery - it is a control plane for delivery that is integrated with key ecosystem services, including Datadog. It receives and processes data inputs to feed user-configurable algorithms, which make traffic routing and load balancing decisions that are communicated into applications via DNS or an HTTP API. Because Openmix delivery optimization is based on the real-time health of infrastructure endpoints, it optimizes user experience; maximizes performance; and provides granular control for organizations according to their unique business rules.
+      <p>
+        <img class="screenshot" src="Images/snapshot.png" />
+      </p>
+    </div>
+    
+    <div class="integration_content"  id="int-configuration">
+      <h2>Configuration</h2>
+      <p>
+        <em>Refer to the <a href="https://portal.cedexis.com/static/help/fusion/html/datadog/datadog.html">Datadog integration documentation<a/> on the Cedexis website.</em>
```

Comment:
Rather than linking to an external docs site, can you please copy the configuration instructions here so that it shows up within the tile, as well as gets populated on to Datadog's own documentation?